src/features/home/components/modules/HomeModule.web.test.tsx

```diff
@@ -80,7 +80,7 @@ describe('<HomeModule />', () => {
 
       // we need to use findAllByLabelText because 'Week-end FRAC' is assigned twice
       // in alt property for image and touchable link
-      expect(await screen.findAllByLabelText('Week-end FRAC')).toBeTruthy()
+      expect(await screen.findAllByLabelText('Week-end FRAC')).not.toHaveLength(0)
```

Comment:
Je ne comprends pas trop ce `not.toHaveLength(0)` ? `toBeInTheDocument` ne fonctionne pas avec `all` je suppose ?